packages/elements/package.json

```diff
@@ -25,8 +25,8 @@
     "release:dev": "node ../../scripts/release/release-dev.js elements reapit-elements-dev",
     "lint:fix": "eslint --cache --ext=ts,tsx,js src --fix",
     "start:dev": "start-storybook -p 6006",
-    "start:prod": "cross-env NODE_ENV=development tsdx watch",
-    "test:ci": "cross-env TZ=UTC jest --ci --colors --coverage --silent --forceExit --detectOpenHandles --runInBand",
+    "start:prod": "serve -l 8080 public/dist",
+    "test:ci": "cross-env TZ=UTC jest --ci --colors --coverage --silent --forceExit",
```

Comment:
Why do you need to change this place?

Reply:
That's the correct command to start storybook after `build:prod`
`tsdx watch` doesn't start anything